File: src/main/java/com/bombproj/model/ProjectDataTable.java
package com.bombproj.model;

import com.queryflow.annotation.Table;
import lombok.Data;

import java.util.Date;

@Data
@Table("project_datatable")
public class ProjectDataTable {

    private String id;
    private String tableName;
    private String tableDesc;
    private Date createTime;
    private Date updateTime;
    private String type;
    private Integer state;
    private String projectId;
    private String userId;
    private Long databaseId;

}

File: src/main/scala/com/bombproj/model/ProjectDataTable.scala
package com.bombproj.model

import java.util.Date

import com.queryflow.annotation.Table

import scala.beans.BeanProperty

@Table("project_datatable")
case class ProjectDataTable() {

    @BeanProperty var id: String = _
    @BeanProperty var tableName: String = _
    @BeanProperty var tableDesc: String = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var updateTime: Date = _
    @BeanProperty var state: Int = _
    @BeanProperty var projectId: String = _
    @BeanProperty var userId: String = _
    @BeanProperty var databaseId: Long = _

}
